.dotnet/src/Generated/Models/UserChatMessage.cs

```diff
@@ -9,13 +9,13 @@ namespace OpenAI.Chat
 {
     public partial class UserChatMessage : ChatMessage
     {
-        internal UserChatMessage(ChatMessageRole role, ChatMessageContent content, IDictionary<string, BinaryData> serializedAdditionalRawData, string participantName) : base(role, content, serializedAdditionalRawData)
+        public UserChatMessage() : base(Chat.ChatMessageRole.User)
```

Comment:
Why is this public ctor generated?

Reply:
Same as https://github.com/joseharriaga/openai-in-typespec/pull/305#discussion_r1868149674

Reply:
Fixed by customization